src/Compilers/CSharp/Test/Syntax/Parsing/RoundTrippingTests.cs

```diff
@@ -600,7 +600,7 @@ public class Class1
     }
 }
 ";
-            ParseAndRoundTripping(text, -1);
+            ParseAndRoundTripping(text, errorCount: 0);
```

Comment:
-1 refers to the previous code causing parse errors.  There are no parse errors with this code anymore.